src/Routes.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { createStackNavigator } from "@react-navigation/stack";
+import { NavigationContainer } from "@react-navigation/native";
+import { Button, Text, View } from 'react-native';
+import { Center } from './Center';
+
+interface RoutesProps{}
+
+const Stack = createStackNavigator();
+
+function Home({ navigation }){
+    return (
+        <Center>
+             <Text>
+                I am home screen
+            </Text>
+            <Button title = "Go to Workout" onPress={() => {
+                navigation.navigate('Workout');
+            }} 
+            />
+        </Center>
+    )
+}
+
+function Workout( {navigation} ){
+    return (
+        <Center>
+             <Text>
+                I am a workout screen
+            </Text>
+            <Button title = "Go to Home" onPress={() => {
+                navigation.navigate('Home')
+            }}
+            />
+        </Center>
+    )
+}
+
+export const Routes: React.FC<RoutesProps> = ({}) => {
```

Comment:
See comment about moving this code to `App.tsx`.

__Note__: after you remove all the components from this file to the suggested locations, this file can be deleted.